clickhouse-native-jdbc/src/test/java/com/github/housepower/jdbc/ConnectionParamITest.java

```diff
@@ -38,6 +40,70 @@ public void init() throws SQLException {
         resetDriverManager();
     }
 
+    @Test
+    public void connectionPatternTest() {
+        String[] jdbcFalse = new String[]{
+                "//ck1:",
+                "//,ck1",
+                "//ck1,ck2,",
+                "//ck1:,ck2",
+                "//ck1,ck2/"
```

Comment:
I have no objection to treating them illegal except the last one, even I think they are recoverable.
Can we make `jdbc:clickhouse//ck1,ck2/` acceptable?

Reply:
It's not a big deal, as long as document it clearly.